### PR TITLE
docs: Updating Wizard/Template information

### PR DESCRIPTION
**Pull request by nickrandolph**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 7abcabe</samp>

This pull request improves the documentation of the Uno Platform app template and its features by consolidating and updating the existing content into a new file `doc/articles/get-started-solution.md`. It also renames and revises the file `doc/articles/uno-app-solution-structure.md` and moves some content from other files to the new file. It updates the table of contents file `doc/articles/toc.yml` accordingly.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/getting-started/wizard/authentication.md [hunk 1]
```diff
@@ -2,32 +2,56 @@
...
 
 Using Uno's authentication tools you can easily integrate in your app a login screen and authenticate users. It also helps you manage and track the state of the app's authentication, by allowing you to log out or switch users as well as other authentication-related actions.
 
 Uno.Extensions.Authentication currently supports these types of authentication:
 
 
-- **None**  
-    No authentication added to the application.
+- #### None  
+    No authentication added to the application. This is the default for both blank and recommended presets.
```
[the remark this answers is not in the document]
Comment (nickrandolph) [reply doc/articles/getting-started/wizard/authentication.md:14]:
```suggestion
    No authentication is added to the application. This is the default for both blank and recommended presets.
```

**Comment by jeromelaban** [comment]
@mergifyio rebase

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.0

**Review by jeromelaban — APPROVED** [review]

**Review by MartinZikmund — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio squash

**Pull request merged**